gradlew

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+
+##############################################################################
+##
+##  Gradle start up script for UN*X
+##
+##############################################################################
+
+# Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
+DEFAULT_JVM_OPTS=""
```

Comment:
Is this auto generated? Is this something to read and understand?

Reply:
(The whole fille)

Reply:
Auto generated.

Reply:
👍